static/src/javascripts/projects/commercial/modules/third-party-tags.js

```diff
@@ -103,8 +103,8 @@ const insertScripts = (
 const loadOther = (): void => {
     const advertisingServices: Array<ThirdPartyTag> = [
         remarketing(),
-        permutive,
-        ias,
+        permutive({ shouldRun: config.get('switches.permutive', false) }),
+        ias({ shouldRun: config.get('switches.iasAdTargeting', false) }),
```

Comment:
following on above suggestion here we could `map` the thirdPartyTags to pass in the switch condition or something similar

Reply:
Alternatively you pass the `switches` object to it?

https://github.com/guardian/commercial-core/blob/b73a58b54042f031abd2156c0f9d038525e55ba3/src/types.ts#L6